WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -423,28 +399,6 @@ extension NotificationDetailsViewController {
     }
 }
 
-// MARK: - Reply View Helpers
-//
-extension NotificationDetailsViewController {
-    func attachReplyViewIfNeeded() {
-        guard shouldAttachReplyView else {
-            replyTextView.removeFromSuperview()
-            return
-        }
-
-        stackView.addArrangedSubview(replyTextView)
-    }
-
-    var shouldAttachReplyView: Bool {
-        // Attach the Reply component only if the notification has a comment, and it can be replied to.
-        //
-        guard let block: FormattableCommentContent = note.contentGroup(ofKind: .comment)?.blockOfKind(.comment) else {
-            return false
-        }
-        return block.action(id: ReplyToCommentAction.actionIdentifier())?.on ?? false
-    }
```

Comment:
Is this variable always `false`? If not, `replyTextView` may appear in the UI and is now removed from this PR.